extensions/hp-extension/src/hpHeart.ts

```diff
@@ -0,0 +1,245 @@
+export default {
+  id: 'LV Function Heart',
+  hasUpdatedPriorsInformation: false,
+  name: 'Default',
+  createdDate: '2021-02-23T19:22:08.894Z',
+  modifiedDate: '2021-02-23T19:22:08.894Z',
+  availableTo: {},
+  editableBy: {},
+  protocolMatchingRules: [
+    {
+      id: 'LVFunction',
+      weight: 150,
+      attribute: 'StudyDescription',
+      constraint: {
+        contains: 'LV Function',
+      },
+      required: true,
+    },
+  ],
+  toolGroupIds: ['default'],
+  stages: [
+    {
+      id: 'hYbmMy3b7pz7GLiaT',
+      name: 'default',
+      viewportStructure: {
+        type: 'grid',
+        properties: {
+          rows: 2,
+          columns: 2,
+        },
+      },
+      displaySets: [
+        {
+          id: 'displaySetCineLax4',
+          // Matches displaysets, NOT series
+          seriesMatchingRules: [
```

Comment:
The series matching rules say to prefer a series that contains CINE, LAX and 4Ch  (then 3Ch, 2Ch)